django_project/cplus_api/tasks/runner.py

```diff
@@ -44,7 +44,23 @@ def run_scenario_analysis_task(scenario_task_id):
     import processing  # noqa
     from processing.core.Processing import Processing
     Processing.initialize()
+
     analysis_task = create_scenario_task_runner(scenario_task)
+
+    # set sieve mask path, snap layer path, and mask layers path
+    from cplus.utils.conf import settings_manager, Settings
+    settings_manager.set_value(
```

Comment:
please use the TaskConfig to get the sieve_mask_path, snap_layer path, mask_layers_paths